h/services/bulk_executor/_executor.py

```diff
@@ -16,7 +16,7 @@
 class BulkExecutor(Executor):
     """Executor of command objects which will modify the database in bulk."""
 
-    def __init__(self, db, authority="lms.hypothes.is"):
```

Comment:
I guess we're removing the default value because the `authority` argument is always passed now. (Whereas previously it was _never_ passed so it seems this didn't need to be any argument.)

Reply:
Nope it didn't, but it made today's job a lot easier.